portfolio/src/main/webapp/dairy.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Antonio's Analytics</title>
+    <script src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="dairyChartScript.js"></script>
```

Comment:
Adding `defer` to these scripts will prevent blocking the HTML document parsing, though the difference might not be noticeable.